.travis.yml

```diff
@@ -6,6 +6,7 @@ rvm:
   - "2.4.0"
 before_install: gem update --system
 script:
+  - RUBY_ENV=travis
```

Comment:
You don't need this. You can use `TRAVIS=true` which already exists on all travis builds: https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables